mslice/models/slice/mantid_slice_algorithm.py

```diff
@@ -90,21 +90,44 @@ def _compute_slice_nonPSD(self, workspace, x_axis, y_axis, norm_to_one):
             plot_data = np.transpose(plot_data)
         return plot_data
 
-    def compute_boltzmann_dist(self, sample_temp, e_axis):
+    def axis_values(self, axis, reverse=False):
+        """
+        Compute a set of bins for the given axis values
+        :param axis: Axis object defining axis details
+        :param reverse: If true then the axis should have values in decreasing order
+        :return: A new numpy array containing the axis values
+        """
+        if reverse:
+            values = np.linspace(axis.end, axis.end, self._get_number_of_steps(axis))
```

Comment:
Should this be `np.linspace(axis.end, axis.start, self._get_number_of_steps(axis))`

Reply:
Yes it should :man_facepalming:. I was trying to be fancy with numpy at one point and put it back wrong. Thanks for spotting that!